get_github_coderabbit_review/cicd/github_actions.py

```
@@ -0,0 +1,818 @@
+            step["run"] = self.run
+        if self.uses:
+            step["uses"] = self.uses
+        if self.with_:
+            step["with"] = self.with_
+        if self.env:
+            step["env"] = self.env
+        if self.if_condition:
+            step["if"] = self.if_condition
+        if self.continue_on_error:
+            step["continue-on-error"] = self.continue_on_error
+        # Note: GitHub Actions では step レベルの timeout は未対応（job レベルのみ）
```

Ruff (0.13.1)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)